crates/precompile/Cargo.toml

```diff
@@ -8,6 +8,9 @@ name = "revm-precompile"
 repository = "https://github.com/bluealloy/revm"
 version = "2.0.3"
 
+# Don't need to run build script outside of this repo
+exclude = ["build.rs", "src/blob/kzg_settings/*.txt"]
```

Comment:
I see the idea and I like the script as we get a very clean and compact variable that we can load and use, but this feels better to be moved to `revme` as a helper command.

We can make an issue for this and do it latter if that is okay with you?

Reply:
I don't see why not keep the build script? It's ran when the files change so you don't have to remember/need a CI job to remind you etc.

Reply:
`.bin` file would stay, would rename it to `.hex` or even `.txt` as it is still readable, and would be used only to store mainnet setup in compact form, this is not something that is expected to be changed at all in future so having build script for it is a overkill. This procedure should be just a helper function inside `revm` and it would be exposed in `revme` if needed to be called in cli.

Reply:
issue: https://github.com/bluealloy/revm/issues/686

Reply:
Alright I guess, but the bin files are not human readable: they're the raw `hex::decode`d bytes of the G1 and G2 points from the trusted setup file

Reply:
How do we want to move forward with this? Seeing we still have the build.rs file